builds/jazz_azure-create-service/components/function/storageblobHandler.js

```diff
@@ -1,9 +1,10 @@
 
 async function create(data, client){
   let output;
-  const blobList = await client.blobContainers.list(data.resourceGroupName, data.appName, data.resourceName);
+  const blobList = await client.blobContainers.list(data.resourceGroupName, data.appName);
```

Comment:
Is resourceName no longer required?

Reply:
after the migration to @azure/arm-* packages, resourceName is not required here as it was throwing error with this parameter